### PR TITLE
Add ActiveSupport section

### PR DESCRIPTION
**Pull request by syndbg**
## Motivation:

Depend as little as possible on `ActiveSupport`.

## Change: 

Added tips about usage of `ActiveSupport` extensions:

* `try!`,
* `starts_with`,
* `ends_with`,
* `in?`,
* `forty_two`,
* `inquiry`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -986,6 +987,22 @@ your application.
   of [sidekiq](https://github.com/mperham/sidekiq) gem.
 <sup>[[link](#background-email)]</sup>
 
+
+## Active Support Core Extensions
+
+* <a name="try-bang"></a>
+  Prefer Ruby 2.3.0's safe navigation operator `&.` rather than `ActiveSupport#try!`.
```
Comment (bbatsov) [inline README.md]: rather than -> over

**Comment by bbatsov** [comment]
I fill this section should be full of advice not to use those - especially the alias methods (e.g. `includes?`, `ends_with?`, etc).

File: README.md (hunk 1 above)
Comment (syndbg) [inline README.md]: :+1:

**Comment by syndbg** [comment]
@bbatsov I'll add them. Don't merge

**Comment by syndbg** [comment]
@bbatsov Added more than I expected I would.

File: README.md [hunk 2]
```diff
@@ -986,6 +987,91 @@ your application.
...
+```ruby
+# bad
+(1..50).to_a.forty_two
+
+# good
+(1..50).to_a[41]
+```
+
+* <a name="inquiry"></a>
+  Prefer Ruby's comparison operators over ActiveSupport's `Array#inquiry`, `Numeric#inquiry' and `String#inquiry`
```
Comment (ausmith) [inline README.md]: Ended up with a single quote instead of a backtick at the end of `Numeric#inquiry`

File: README.md [hunk 3]
```diff
@@ -986,6 +987,91 @@ your application.
...
+
+# good
+ruby = 'two'
+ruby == 'two'
+
+# bad - Array#inquiry
+pets = %w(cat dog).inquiry
+pets.gopher?
+# good
+pets.include? 'cat'
```
Comment (ausmith) [inline README.md]: It feels self explanatory, but you didn't setup the `pets` object for your good example and it's still using the `inquiry` version.

**Comment by ausmith** [comment]
Minus my nitpicks, I like this change. :+1:

**Comment by syndbg** [comment]
Corrected.

**Comment by syndbg** [comment]
@bbatsov 2.3 squiggly heredocs tip here -> https://github.com/bbatsov/ruby-style-guide/pull/523

The other comments are also addressed.


Btw, I was under the impression that markdown headers should have 2 lines between them and other content.

**Comment by bbatsov** [comment]
> Btw, I was under the impression that markdown headers should have 2 lines between them and other content.

Nope. One line is just fine. I think this will work even without blank lines, but this is pretty hard to read.

**Pull request merged**